rele/client.py

```diff
@@ -49,6 +50,22 @@ def __init__(self):
                 credentials=settings.RELE_GC_CREDENTIALS)
 
     def publish(self, topic, data, blocking=False, **attrs):
+        """Publishes data to Pub/Sub adding a timestamp `published_at` to the attrs.
+
+        Usage::
+
+            publisher = Publisher()
+            publisher.publish('topic_name', {'foo': 'bar'})
+
+        :param topic: string topic to publish the data.
+        :param data: dict with the content of the message.
+        :param blocking: boolean, default False.
+        :param attrs: Extra parameters to be published.
+        :return: future
+        """
+
+        attrs['published_at'] = str(time.time())
```

Comment:
Since casting to string is needed, shouldn't we do the opposite on the consuming side?

Reply:
If every attribute in the message has to be a string I don't see the point of doing this. Also it should be implemented by the developer that is writing the subscription, isn't it?

Reply:
I think what @tobami is suggesting is that we deserialize the message data from bytes to a python dictionary. So we may want to serialize from a string timestamp to a python datetime object. Is that right?

Reply:
Yep, same as we don't make the user decode and deserializer every payload, published_at has a very specific use case, so we'll always want it in the right type

Reply:
Okay, just implemented it.

Reply:
@tobami @andrewgy8 Can I merge it?